codeblog/src/components/pages/Uploads/Uploads.jsx

```diff
@@ -1,16 +1,106 @@
-import React from 'react'
+import React from 'react';
+import styles from './Uploads.module.css';
 
 function Uploads() {
   return (
     <>
-        <div>Uploads</div>
-        <div>Uploads</div>
-        <div>Uploads</div>
-        <div>Uploads</div>
-        <div>Uploads</div>
-        <div>Uploads</div>
+      <section
+        className={styles.aboutusSection}
+        id="aboutusSection"
```

Comment:
don't make same id and classname 
put id like 
aboutUsSectionId
and use camel casing in aboutUs